docs/CHANGELOG.rst

```diff
@@ -9,6 +9,8 @@ Current (Unreleased)
 --------------------
 
 - Initial start of the changelog documentation. [`@terrancedejesus <https://github.com/terrancedejesus>`_]
+- Added `pytest` to the project. [`@terrancedejesus <https://github.com/terrancedejesus>`_]
+- Added `test_emulation.py` test file with several test cases for emulation metadata. [`@terrancedejesus <https://github.com/terrancedejesus>`_]
```

Comment:
We should bucket the changelog under version bumps. Which means
* for changes requiring a version bump and targeting `dev`, they would bucket up
* for changes directly to main, they would append the _current_ version